vowpalwabbit/slim/test/ut_util.cc

```diff
@@ -351,7 +351,7 @@ invalid_model_param invalid_model_param[] = {
 
 INSTANTIATE_TEST_SUITE_P(VowpalWabbitSlim, invalid_model_test, ::testing::ValuesIn(invalid_model_param));
 
-TEST(VowpalWabbitSlim, multiclass_data_4)
+TEST(vowpal_wabbit_slim, multiclass_data_4)
```

Comment:
If you change this you also need to update this: https://github.com/VowpalWabbit/vowpal_wabbit/blob/5c920b44f3f554820ff1fce01a7976dd1907814f/.github/workflows/build_vw_slim.yml#L31